homeassistant/components/insteon/fan.py

```diff
@@ -60,9 +63,7 @@ async def async_turn_on(
         **kwargs,
     ) -> None:
         """Turn on the fan."""
-        if percentage is None:
-            percentage = 50
-        await self.async_set_percentage(percentage)
+        await self.async_set_percentage(percentage or 67)
```

Comment:
This will make it no longer possible to turn the fan off by sending `percentage=0`. Is that on purpose?

Reply:
Yes.  It seems like unexpected behavior to turn off when calling turn on.

Reply:
Ok.

We actually have done that for lights with `brightness_step`. If it becomes 0, it will turn off.